TouristPass.java

```diff
@@ -0,0 +1,13 @@
+public class TouristPass extends Pass {
```

Comment:
This file needs a bunch of style changes: spaces between parameters, newlines, etc.